notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        print(f"Prepare work directory for the station: {station_name}")
</pre><p>Lets print work directory e.g. <code>print(f"Prepare work directory {work_dir} for the station: {station_name}")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>